synapse/handlers/admin.py

```diff
@@ -31,7 +31,7 @@ class AdminHandler:
     def __init__(self, hs: "HomeServer"):
         self.store = hs.get_datastores().main
         self.storage = hs.get_storage()
-        self.state_store = self.storage.state
+        self.state_storage = self.storage.state
```

Comment:
oof, I would have loved for these to get underscore prefixes while you were renaming them.